tpv34/ASAGI_file

```diff
@@ -0,0 +1,6 @@
+tpv34_mux-fault.nc: 
+https://syncandshare.lrz.de/getlink/fiPzx6ebB4oor4BjtXvpmM92/tpv34_mux-fault.nc
```

Comment:
This is not good as it is not a permanent link.
Would be better to point to the script you use to generate these files

Reply:
it is quite a big file and need some time to create it. any idea where to put it then?
I will provide a script on the tpv34 cookbook (ongoing).

Reply:
How much time? Why does it take time to generate it?

Reply:
It depends on the number of input points for CVM-H. They have a 1km resolution. 
Anyway, I removed the links already and just directing it to the tpv34 cookbook, see https://github.com/SeisSol/Examples/pull/5/commits/c4dd7d3bab35bf83a163817f2051bca6070958d6